.github/workflows/ci.yml

```diff
@@ -28,20 +28,50 @@ jobs:
 
   unit:
     runs-on: ubuntu-latest
+    services:
+      mysql:
+        image: mysql:8.0
```

Comment:
Is it worth setting up a matrix to test against 8 and 9? And maybe even patch releases like 8.4?